dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -23,7 +23,11 @@ recommonmark >= 0.4.0
 Sphinx
 sphinx_rtd_theme
 sphinx-markdown-tables
+
 # For generating notebooks in documentation
 nbsphinx
 ipython
 ipykernel
+
+# For verifying rst
+rstcheck
```

Comment:
Consider pinning the version here, as we do for other packages that check for correctness like mypy, pylint, yapf, etc. It can be frustrating when a local build and ci disagree because they're using different package versions.

Reply:
that makes total sense, thank you!